### ISSUE TITLE
traffic-management/ingress/ingress-control/ nodeport tab doesn't increment count

### ISSUE THREAD

**Issue by yanrongshi** [istio/istio.io#11962]
### issue
I found an upstream problem during the Chinese localization process, however, no matter what I do locally, the error still exists, I need help。
https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/
#11932 

`content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md`
![image](https://user-images.githubusercontent.com/29862632/191065732-ec80d760-8ad3-49d3-9720-64d9ab315c38.png)
### current
![image](https://user-images.githubusercontent.com/29862632/191066183-2b5a9881-c2dc-483d-a2a0-7f254da634ed.png)

**Comment by ericvn** [comment]
Looks like an indention issue. If you want, I can help within the various PRs.

**Comment by yanrongshi** [comment]
> Looks like an indention issue. If you want, I can help within the various PRs.

Can you help me with this document? #11932

**Comment by ericvn** [comment]
I'll reopen this and fix the Nodeport tab in the English side and that can be synced over.

**Comment by yanrongshi** [comment]
> I'll reopen this and fix the Nodeport tab in the English side and that can be synced over.

Ok, looking forward to your results

**Comment by ericvn** [comment]
Looking back at the various PRs on the ingess-control/index.md, when #7737 was merged, it looked correct, but when #11326 was merged, it was incorrect in terms of numbering.

Looks look if this was a change related to changing the markdown (#11534). With that change, things are incorrectly numbered (https://deploy-preview-11534--preliminary-istio.netlify.app/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports) and before that it worked (https://deploy-preview-11733--preliminary-istio.netlify.app/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports).

So, looks like this is a change related to changing the markdown which was required as the older markdown was removed in recent Hugo releases (has been deprecated for a long time). The lack of doc testing with the 1.15 release most likely lead to that page never being looked at.

That said, we already removed the numbering in the Getting Started page (long ago), so I did not see any numbering problems when I went to the website during my personal looking at pages. This page would have been one I looked at as I know it's a popular page.

I believe we should simply remove the numbering to match the "Getting Started" unless someone determines we really need the numbering and then someone will need to look at the tab and tabset html files (most likely) to fix the issue. I say these files as having the same code outside the tab works (just copied it outside as a test). I suspect it might be related to the
```
                {{- $text := partial "strip_indent.html" (dict "content" .content "pos" .Position) -}}
                {{- $text | markdownify -}}
```
in tabset.html since it seems that the indentation is also wrong. 
<img width="867" alt="Screen Shot 2022-09-19 at 1 19 12 PM" src="https://user-images.githubusercontent.com/10537847/191086550-0533de33-a41a-4518-82e4-2d61ba040ff4.png">

**Comment by yanrongshi** [comment]
> 回头看ingess-control/index.md上的各种PR，#7737合并的时候，看起来是对的，但是#11326合并的时候，在编号上就不对了。
> 
> 看起来这是否是与更改降价相关的更改（#11534）。随着这种变化，事情的编号不正确（[https://deploy-preview-11534--preliminary-istio.netlify.app/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress -ip-and-ports](https://deploy-preview-11534--preliminary-istio.netlify.app/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports)），在此之前它工作（[https://deploy-preview-11733--preliminary-istio.netlify.app/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining -the-ingress-ip-and-ports](https://deploy-preview-11733--preliminary-istio.netlify.app/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports)）。
> 
> 因此，看起来这是与更改降价相关的更改，因为在最近的 Hugo 版本中删除了较旧的降价（已被弃用很长时间）。1.15 版本缺乏文档测试很可能导致该页面从未被查看。
> 
> 也就是说，我们已经删除了“入门”页面中的编号（很久以前），所以我在个人查看页面时访问该网站时没有发现任何编号问题。这个页面本来是我看过的，因为我知道它是一个受欢迎的页面。
> 
> 我相信我们应该简单地删除编号以匹配“入门”，除非有人确定我们确实需要编号，然后有人需要查看选项卡和选项卡集 html 文件（很可能）来解决问题。我说这些文件在选项卡之外具有相同的代码有效（只是将其复制到外部作为测试）。我怀疑这可能与
> 
> ```
>                 {{- $text := partial "strip_indent.html" (dict "content" .content "pos" .Position) -}}
>                 {{- $text | markdownify -}}
> ```
> 
> 在 tabset.html 中，因为缩进似乎也是错误的。 <img alt="屏幕截图 2022-09-19 在 1 19 12 PM" width="867" src="https://user-images.githubusercontent.com/10537847/191086550-0533de33-a41a-4518-82e4-2d61ba040ff4.png">

Numbering may not seem that important, but if we don't provide numbering capabilities here, do we break the unity of the document
